nautobot/utilities/templatetags/helpers.py

```diff
@@ -212,49 +320,102 @@ def has_one_or_more_perms(user, permissions_list):
     return False
 
 
+@library.filter()
 @register.filter()
 def split(string, sep=","):
-    """
-    Split a string by the given value (default: comma)
+    """Split a string by the given value (default: comma)
+
+    Args:
+        string (str): string to split into a list
+        sep (str default=,): separator to look for in the string
+
+    Returns:
+        [list]: List of string, if the separator wasn't found, list of 1
     """
     return string.split(sep)
 
 
+@library.filter()
 @register.filter()
 def as_range(n):
-    """
-    Return a range of n items.
+    """Return a range of n items.
+
+    Args:
+        n (int, str): Number of element in the range
+
+    Returns:
+        [list, Range]: range function from o to the value provided. Returns an empty list if n is not valid.
+
+    Example:
+        {% for i in record.parents|as_range %}
+            <i class="mdi mdi-circle-small"></i>
+        {% endfor %}
     """
     try:
         int(n)
-    except TypeError:
+    except (TypeError, ValueError):
         return list()
-    return range(n)
+    return range(int(n))
 
 
+@library.filter()
 @register.filter()
 def meters_to_feet(n):
-    """
-    Convert a length from meters to feet.
+    """Convert a length from meters to feet.
+
+    Args:
+        n (int, float, str): Number of meters to convert
+
+    Returns:
+        [float]: Value in feet
     """
     return float(n) * 3.28084
 
 
-@register.filter
+@library.filter()
+@register.filter()
 def get_item(d, key):
+    """Access a specific item/key in a dictionary
+
+    Args:
+        d (dict): dictionary containing the data to access
+        key (str]): name of the item/key to access
+
+    Returns:
+        [any]: Value of the item in the dictionary provided
+
+    Example:
+        >>> get_items(data, key)
+        "value"
+    """
     return d.get(key)
 
 
-@register.filter
+@library.filter()
+@register.filter()
 def settings_or_config(key):
+    """Get a value from Django settings (if specified there) or Constance configuration (otherwise)."""
     return get_settings_or_config(key)
 
 
+@register.filter
```

Comment:
```suggestion
@library.filter()
@register.filter()
```

We also need to add a couple unit test cases for this filter now that we have tests for the other ones in this file. I'm fine with deferring that for now though.